src/components/Button/index.tsx

```diff
@@ -176,23 +176,27 @@ export const ButtonOutlined = styled(BaseButton)`
   }
 `
 
-export const ButtonYellow = styled(BaseButton)`
-  background-color: ${({ theme }) => theme.deprecated_yellow3};
-  color: white;
+export const ButtonYellow = styled(BaseButton)<{ redesignFlag?: boolean }>`
+  background-color: ${({ theme, redesignFlag }) => (redesignFlag ? theme.accentWarningSoft : theme.deprecated_yellow3)};
+  color: ${({ theme, redesignFlag }) => (redesignFlag ? theme.accentWarning : 'white')};
   &:focus {
-    box-shadow: 0 0 0 1pt ${({ theme }) => darken(0.05, theme.deprecated_yellow3)};
-    background-color: ${({ theme }) => darken(0.05, theme.deprecated_yellow3)};
+    box-shadow: ${({ theme, redesignFlag }) => !redesignFlag && `0 0 0 1pt ${theme.deprecated_yellow3}`};
+    background-color: ${({ theme, redesignFlag }) =>
+      redesignFlag ? theme.accentWarningSoft : darken(0.05, theme.deprecated_yellow3)};
   }
   &:hover {
-    background-color: ${({ theme }) => darken(0.05, theme.deprecated_yellow3)};
+    background-color: ${({ theme, redesignFlag }) =>
+      redesignFlag ? darken(0.08, theme.accentWarningSoft) : darken(0.05, theme.deprecated_yellow3)};
```

Comment:
probably want to avoid the darken here?

Reply:
clarified how to implement the design css, here is the link to figma: https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-App-Spec-(Phase-0-%26-Phase-1)-%5BWIP%5D?node-id=477%3A80743
should be a background overlay. its the hover yellow warning button state.